src/pretix/control/forms/organizer.py

```diff
@@ -25,9 +26,20 @@ def clean_slug(self):
 
 
 class OrganizerUpdateForm(OrganizerForm):
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['slug'].widget.attrs['readonly'] = 'readonly'
 
     def clean_slug(self):
         return self.instance.slug
+
+
+class OrganizerSettingsForm(SettingsForm):
+
+    logo_image = ExtFileField(
```

Comment:
Here's a subtle thing that I also did not notice during review and that caught me by surprise when I tested your changes: Our settings key-value store is hierarchical. By this I mean that if you try to access a setting of an event and the setting is not set for that event, the setting of the organizer will be accessed instead. If that does not exist, the global setting will be taken and only if this also does not exist, the hardcoded default will be used.

Currently, we rarely use this hierarchy, as it is hard to represent it in the user interface and normally it does not gain any attention as the different levels normally use different keys. Here, however, the effect is that if you set the organizer logo, all events without a logo of their own will inherit the logo. I believe this is not what a user expects, or at least I did not expect that. The fix is easy: Call this thing ``organizer_logo_image`` instead :wink: